media/js/wiki.js

```diff
@@ -302,6 +302,8 @@
         var $pageButtons = $('.page-buttons');
         var pageButtonsOffset = $pageButtons.offset();
 
+        var stickyFeatureEnabled = $pageButtons.attr('data-sticky') === 'true';
```

Comment:
I don't think we need to waffle it. We test it, we like the results, we're rolling it into production.